spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -15,6 +15,7 @@
     given_I_am_a_GDS_editor
     setup_publishing_api
     stub_default_publishing_api_put_intent
+    stub_link_checking_for_default_fixtures
```

Comment:
Is this needed? The `link_report` factory can be used to create a link_report against a step.